torch/csrc/Module.cpp

```diff
@@ -445,9 +445,19 @@ PyObject *THPModule_addDocStr(PyObject *_unused, PyObject *args)
           "method '%s' already has a docstring", m->d_method->ml_name);
     }
     m->d_method->ml_doc = doc_str;
-  } else {
-    return PyErr_Format(PyExc_TypeError,
-        "don't know how to add docstring to type '%s'", Py_TYPE(obj)->tp_name);
+  } else {  // e.g. python function
```

Comment:
I think we should document Python functions in Python using standard docstrings

Reply:
Sure, I removed it. I thought maybe keeping every torch.* doc at the same file would be easier to organize.